ecs-server.py

```diff
@@ -168,7 +168,7 @@ def is_valid_docker_image(image_name):
 
 
 def run_docker_task(image_name, ecs_task_env, environment_overrides):
-    cmd = ["docker", "run", "--detach"]
+    cmd = ["docker", "run", "--interactive"]
```

Comment:
That is not going to work for `dpnt-coverage` as the test spins up multiple containers concurrently (one for each challenge round). I use `docker logs` to inspect a stopped container, maybe we can document that.